<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Widget</class>
    <widget class="QWidget" name="Widget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>612</width>
                <height>907</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Widget</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>531</width>
                    <height>150</height>
                </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
                <item row="2" column="2">
                    <spacer name="horizontalSpacer">
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>40</width>
                                <height>20</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item row="9" column="1">
                    <widget class="QLabel" name="label">
                        <property name="text">
                            <string>TextLabel</string>
                        </property>
                    </widget>
                </item>
                <item row="6" column="0">
                    <widget class="QLabel" name="label_5">
                        <property name="text">
                            <string>Frequency</string>
                        </property>
                    </widget>
                </item>
                <item row="6" column="1">
                    <widget class="QSpinBox" name="spinBox">
                        <property name="minimum">
                            <number>1</number>
                        </property>
                        <property name="maximum">
                            <number>200</number>
                        </property>
                        <property name="singleStep">
                            <number>5</number>
                        </property>
                        <property name="value">
                            <number>100</number>
                        </property>
                    </widget>
                </item>
                <item row="3" column="0">
                    <widget class="QPushButton" name="pushButton_2">
                        <property name="maximumSize">
                            <size>
                                <width>100</width>
                                <height>100</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>Stop</string>
                        </property>
                    </widget>
                </item>
                <item row="9" column="0">
                    <widget class="QPushButton" name="pushButton_3">
                        <property name="maximumSize">
                            <size>
                                <width>100</width>
                                <height>100</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>Exit</string>
                        </property>
                    </widget>
                </item>
                <item row="2" column="0">
                    <widget class="QPushButton" name="pushButton">
                        <property name="maximumSize">
                            <size>
                                <width>100</width>
                                <height>100</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>Start</string>
                        </property>
                    </widget>
                </item>
                <item row="10" column="0">
                    <spacer name="verticalSpacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>40</height>
                            </size>
                        </property>
                    </spacer>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>160</y>
                    <width>531</width>
                    <height>431</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="my_widget" name="my_widget" native="true">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <property name="minimumSize">
                            <size>
                                <width>100</width>
                                <height>100</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>1000</width>
                                <height>1000</height>
                            </size>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QTextBrowser" name="textBrowser">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>590</y>
                    <width>531</width>
                    <height>91</height>
                </rect>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>690</y>
                    <width>100</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="maximumSize">
                <size>
                    <width>100</width>
                    <height>100</height>
                </size>
            </property>
            <property name="text">
                <string>get directory</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>720</y>
                    <width>100</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="maximumSize">
                <size>
                    <width>100</width>
                    <height>100</height>
                </size>
            </property>
            <property name="text">
                <string>get file</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>120</x>
                    <y>690</y>
                    <width>421</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>TextLabel</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_3">
            <property name="geometry">
                <rect>
                    <x>120</x>
                    <y>720</y>
                    <width>421</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>TextLabel</string>
            </property>
        </widget>
    </widget>
    <customwidgets>
        <customwidget>
            <class>my_widget</class>
            <extends>QWidget</extends>
            <header>my_widget.h</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
